modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/form.js

```diff
@@ -14,7 +14,7 @@ AUI.add(
 		};
 
 		var number = function(val, node, ruleValue) {
-			var regex = /^[+\-]?(\d+)(\.\d+)?([eE][+-]?\d+)?$/;
+			var regex = /^[+\-]?(\d+)([.|,]\d+)*?([eE][+-]?\d+)?$/;
```

Comment:
Hi @omarzzn3 ,

I think, that we do not need "*?". Probably "*" is enough. What do you think?